docs/system-architecture.md

```diff
@@ -0,0 +1,193 @@
+# EPP Evaluation Tracker System Architecture
+
+## System Diagram
+
+In the following diagram, "FE" means "front end" and "BFF" means backend for
+frontend. Micro data store is a small database used to support the BFF. Each of
+the components is described in more detail below.
+
+```mermaid
+flowchart LR
+    A(fa:fa-users) -->|via browser| B["`EPPETA *FE*`"]
+    B -->|fa:fa-times-circle https| C[ODS-API]
```

Comment:
Interestingly, GitHub does not show the font-awesome integration from Mermaid (`fa:fa-times-circle`). Thus you can't see that there's a big X there. I should modify that. You also can't see that the first node starts with a users icon.

Reply:
```suggestion
    B -->|X https| C[ODS-API]
```